app/components/post-search-results-affiliate-item.js

```diff
@@ -1,15 +1,23 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 
 import { trackActions, trackAffiliateUnit } from '../utils/track';
 import { linkToProxyLink } from '../utils/affiliate';
 
 export default Component.extend({
+  wikiVariables: service(),
+
   isInContent: false,
+  articleId: null,
 
   classNames: ['post-search-results-aff-item'],
 
-  getUnitLink: computed('affiliateUnit', 'isInContent', function () {
+  wikiId: reads('wikiVariables.id'),
+
+  getUnitLink: computed('affiliateUnit', 'isInContent', 'articleId', function () {
```

Comment:
can you add `wikiId` to the list here?

Reply:
https://github.com/Wikia/mobile-wiki/pull/2157/commits/7015cc7d8e6db4aad57d49ffae757d0e8cfc78a3